collections/ansible_collections/cloudkit/templates/roles/ocp_virt_vm/tasks/delete.yaml

```diff
@@ -0,0 +1,123 @@
+---
+- name: Set VM facts
+  ansible.builtin.set_fact:
+    vm_name: "{{ template_parameters.vm_name | default(vm_order.metadata.name) }}"
+    vm_namespace: "{{ template_parameters.vm_namespace | default(vm_target_namespace) }}"
+    additional_disks: "{{ template_parameters.additional_disks | default([]) }}"
+    vm_expose_service: "{{ template_parameters.vm_expose_service | default(false) }}"
+
+- name: Stop VirtualMachine
+  kubernetes.core.k8s:
+    api_version: kubevirt.io/v1
+    kind: VirtualMachine
+    name: "{{ vm_name }}"
+    namespace: "{{ vm_namespace }}"
+    state: present
+    definition:
+      spec:
+        running: false
+  ignore_errors: true
+
+- name: Wait for VM to stop
+  kubernetes.core.k8s_info:
+    api_version: kubevirt.io/v1
+    kind: VirtualMachine
+    name: "{{ vm_name }}"
+    namespace: "{{ vm_namespace }}"
+    wait: true
+    wait_condition:
+      type: Ready
+      status: "False"
+    wait_timeout: 300
+  ignore_errors: true
```

Comment:
I don't think we need to stop it, I expect kubevirt to do it for us when the CR is deleted